tests/fixtures/add/local/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
Can you rename this to `Cargo.toml.sample`? We renamed all the other `Cargo.toml` files in the `fixtures` directory because it was confusing `cargo install`. Thanks!